### DISCUSSION TITLE
How does the update cycle work in Min?

### DISCUSSION THREAD

**Discussion by ghost** [minbrowser/min#2282] [Q&A] [answered]
Hi, 

I would like to know who is the developer responsible for the Min open project. 
1. Who is the developer responsible for the Min open project? 
2. What are the criteria for accepting future pull-requests in Min open project? 
3. How does the update cycle work in Min open project?  Is the update monthly or yearly in the open project Min?

*If anyone answers any questions, I'm very happy ^^*

**Comment by PalmerAL** [comment, marked as answer]
It's just a hobby project by me. There aren't very specific criteria; for feature requests it's a combination of the usefulness of the feature / maintenance cost / how well it fits with the overall design of Min. Likewise, I try to make updates after a certain amount of development has occurred, while also keeping in mind the Electron update schedule.
